pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -413,9 +413,14 @@ func (lbaas *LbaasV2) EnsureLoadBalancer(clusterName string, apiService *api.Ser
 		for _, host := range hosts {
 			addr, err := getAddressByName(lbaas.compute, host)
 			if err != nil {
-				// cleanup what was created so far
-				_ = lbaas.EnsureLoadBalancerDeleted(clusterName, apiService)
+				if err == ErrNotFound {
+					// Node failure, do not create member
```

Comment:
should log warning for each skipped node, log message should contain host in which address was not found.